pkg/slo-controller/noderesource/noderesource.go

```diff
@@ -135,7 +135,7 @@ func (r *NodeResourceReconciler) isGPUResourceNeedSync(new, old *corev1.Node) bo
 }
 
 func (r *NodeResourceReconciler) updateGPUNodeResource(node *corev1.Node, device *schedulingv1alpha1.Device) error {
-	if device == nil {
+	if device == nil || len(device.Spec.Devices) == 0 {
 		return nil
 	}
 	memoryTotal := resource.NewQuantity(0, resource.DecimalSI)
```

Comment:
`memoryTotal` should be shown in BinarySI format, and `coreTotal` should be DecimalSI